src/components/DigitalCredentials/CredentialsInfo.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div>
+    <v-card class="mb-3">
+      <v-card-title
+        class="font-weight-bold word-wrap"
+      >
+        Where to use your Business Card
+      </v-card-title>
+      <v-card-text>
+        <p class="mb-2">
+          In the near future you will be able to use your business card credeptial to obtain a business licence,
+          business bank account, business permits. You can also get access to services like the forest client
+          application.
+        </p>
+        <a
+          class="text-decoration-none"
+          href=""
+          target="_blank"
+        >Signup to be part of the business licence credential pilot </a>
```

Comment:
"Sign up" ?

Should any of these apps be proper nouns and capitalized?

Is this still called a pilot?

Reply:
Looks good.